cfpb/consumerfinance.gov
# Fix lint warnings in JavaScript tests

Fix lint warnings with:
- camelcase
- max-len
- no-console
- no-shadow
- no-unused-vars
- prefer-const

Examples of warning:
- This line has a length of N. Maximum allowed is 80  max-len
- 'chartMenu' is already declared in the upper scope  no-shadow
- 'chartMenu' is assigned a value but never used      no-unused-vars
- 'discountEl' is never reassigned. Use 'const' instead  prefer-const
- Unexpected console statement  no-console
- Identifier 'counseling_agencies' is not in camel case  camelcase
- Identifier 'html_url' is not in camel case  camelcase

## Additions

- eslint-disable-next-line no-unused-vars
- eslint-disable-next-line no-console
- eslint-disable-next-line camelcase

## Removals

- unused variables


## Changes

- split lines longer than 80 characters onto multiple lines
- variable use of const


## How to test this PR

1. gulp lint:tests
2. gulp test:unit --ci --headless
3. Verify no tests start to fail after fixing lint warning


## Notes and todos

Fix lint warnings for:
- Unexpected 'todo' comment  no-warning-comments

## Checklist



  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance

### Front-end testing



#### Browser testing

Visually tested in the following supported browsers:



#### Accessibility


#### Other
